tests/framework/ab_test.py

```diff
@@ -212,13 +164,24 @@ def git_ab_test_guest_command(
     """The same as git_ab_test_command, but via SSH. The closure argument should setup a microvm using the passed
     paths to firecracker and jailer binaries."""
 
-    def test_runner(firecracker, jailer):
+    def test_runner(workspace_dir):
+        bin_dir = get_binary(
+            "firecracker", workspace_dir=workspace_dir
+        ).parent.resolve()
+        firecracker, jailer = bin_dir / "firecracker", bin_dir / "jailer"
         microvm = microvm_factory(firecracker, jailer)
         return microvm.ssh.run(command)
 
-    (_, old_out, old_err), (_, new_out, new_err), the_same = git_ab_test_with_binaries(
-        test_runner, comparator, a_revision=a_revision, b_revision=b_revision
-    )
+    dir_a = git_clone(Path("build") / a_revision, a_revision)
+    res_a = _, old_out, old_err = test_runner(dir_a)
+    if b_revision:
+        dir_b = git_clone(Path("build") / b_revision, b_revision)
+    else:
+        # By default, pytest execution happens inside the `tests` subdirectory. Pass the repository root, as
+        # documented.
+        dir_b = Path.cwd().parent
+    res_b = _, new_out, new_err = test_runner(dir_b)
+    the_same = comparator(res_a, res_b)
```

Comment:
```suggestion
    (_, old_out, old_err), (_, new_out, new_err), the_same = git_ab_test(
        test_runner, comparator, a_revision=a_revision, b_revision=b_revision
    )
```

together with an (unused) `_is_a: bool` parameter on `test_runner`.

Reply:
Done. Thanks!